src/briefcase/integrations/linuxdeploy.py

```diff
@@ -61,22 +65,26 @@ def install(self):
             linuxdeploy_appimage_path = self.command.download_url(
                 url=self.linuxdeploy_download_url, download_path=self.command.tools_path
             )
-            self.command.os.chmod(linuxdeploy_appimage_path, 0o755)
-            self.patch_elf_header()
         except requests_exceptions.ConnectionError as e:
             raise NetworkFailure("downloading linuxdeploy AppImage") from e
 
-    def upgrade(self):
-        """Upgrade an existing linuxdeploy install."""
-        if self.exists():
-            self.command.logger.info("Removing old LinuxDeploy install...")
+        with self.command.input.wait_bar("Installing linuxdeploy..."):
```

Comment:
This description isn't entirely true....but I think it provides better consistency with the other tools (rather than "post processing linuxdeploy install" for instance) and does communicate to the user a task related to the installation after downloading is happening.